server/services/starUpgrade.js

```diff
@@ -350,12 +352,12 @@ module.exports = class StarUpgradeService extends EventEmitter {
             })
             .map(s => {
                 const effectiveTechs = this.technologyService.getStarEffectiveTechnologyLevels(game, s);
-                const terraformedResources = this.starService.calculateTerraformedResources(s.naturalResources, effectiveTechs.terraforming);
+                let terraformedResources = this.starService.calculateTerraformedResources(s.naturalResources, effectiveTechs.terraforming);
```

Comment:
`let terraformedResources = this.starService.calculateTerraformedResources(s.naturalResources[infrastructureType], effectiveTechs.terraforming);`?

Reply:
Why not :stuck_out_tongue_winking_eye: 
Changed back to const